binary_transparency/firmware/cmd/ftmap/README.md

```diff
@@ -1,7 +1,26 @@
 Verifiable Maps
 ===============
 
-This is a work in progress.
+The verifiable map in this firmware transparency demonstrates how annotations (also known as attestations) about a piece of firmware can be verifiably presented to clients.
+The naive way of doing this would be to simply give the client inclusion proofs to the annotations in the log.
+The problem with this approach is that there is no way to prove to the client that they have been presented *all* relevant annotations; logs do not support non-inclusion proofs.
```

Comment:
nit: "proofs of non-inclusion" (a consistency proof is technically a "non-inclusion proof")